.changeset/tiny-wasps-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/guideline-blocks-settings": patch
+---
+
+fix(LinkSelector): bug fixes and qol changes
```

Comment:
```suggestion
fix(LinkSelector): initial click triggered twice
```
What do you think? 
fix(): bug fix is pretty odd as it says both times the same

Reply:
Sure!